src/NLog/Layouts/Log4JXmlEventLayout.cs

```diff
@@ -48,8 +48,6 @@ namespace NLog.Layouts
     /// </remarks>
     [Layout("Log4JXmlEventLayout")]
     [ThreadAgnostic]
-    [ThreadSafe]
-    [AppDomainFixedOutput]
```

Comment:
Is the removal of `[AppDomainFixedOutput]` on purpose? 

(looks like a copy-paste...euh..remove error)

Reply:
It is on purpose, because it makes no sense to have `[AppDomainFixedOutput]` for `Log4JXmlEventLayout`. As it would mean static output always.

Reply:
ok!